Userland/Libraries/LibC/signal.h

```diff
@@ -6,9 +6,9 @@
 
 #pragma once
 
-// Includes essentially mandated by POSIX:
 // https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/signal.h.html
-#include <time.h>
+// The <signal.h> header shall define the timespec structure as described in <time.h>.
+#include <Kernel/API/POSIX/time.h>
```

Comment:
https://github.com/SerenityOS/serenity/commit/17840038f2b32b4963f9f2d4d7e58649b6bbfab6 was about this recently too (@spholz)